instances/widgets.treasury-factory.near/widget/pages/settings/MembersEditor.jsx

```diff
@@ -194,6 +204,45 @@ function cleanInputs() {
   setRoles([]);
 }
 
+function detectImpactedPermissions() {
+  const { updatedPolicy } = updateDaoPolicy(
+    new Map(roles.map((role) => [role.title, role.value]))
+  );
+  const permissionsImpactions = updatedPolicy.roles.map((role) => {
+    const votersSize = role.kind.Group.length;
+    const defaultVotersSize = daoPolicy.roles.find((r) => r.name === role.name)
+      .kind.Group.length;
+    const threshold = Object.values(role.vote_policy)[0].threshold;
```

Comment:
it's possible that there is no role specific`vote_policy` and it's uses `default_vote_policy`, in case of existing daos, please add that check, [example](https://github.com/NEAR-DevHub/neardevhub-treasury-dashboard/blob/staging/instances/widgets.treasury-factory.near/widget/lib/common.jsx#L32)